### PR TITLE
mgmt, postgresqlhsc, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to [issue#35316](https://github.com/Azure/azure-sdk-for-java/issues/35316)

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/cosmosdbforpostgresql/azure-resourcemanager-cosmosdbforpostgresql/src/test/java/com/azure/resourcemanager/cosmosdbforpostgresql/CosmosDBForPostgreSqlManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                .withEnableHa(false)
+                .withCoordinatorServerEdition("GeneralPurpose")
+                .withNodeServerEdition("MemoryOptimized")
+                .withCoordinatorStorageQuotaInMb(131072)
+                .withNodeStorageQuotaInMb(524288)
+                .withCoordinatorVCores(2)
+                .withNodeVCores(4)
+                .withCoordinatorEnablePublicIpAccess(true)
+                .withNodeEnablePublicIpAccess(true)
+                .withNodeCount(0)
```
Comment (XiaofeiCao) [inline sdk/cosmosdbforpostgresql/azure-resourcemanager-cosmosdbforpostgresql/src/test/java/com/azure/resourcemanager/cosmosdbforpostgresql/CosmosDBForPostgreSqlManagerTests.java:101]: What does nodeCount=0 mean?

Comment (v-hongli1) [reply sdk/cosmosdbforpostgresql/azure-resourcemanager-cosmosdbforpostgresql/src/test/java/com/azure/resourcemanager/cosmosdbforpostgresql/CosmosDBForPostgreSqlManagerTests.java:101]: When node count is 0, it represents a single node configuration with the ability to create distributed tables on that node. 2 or more worker nodes represent multi-node configuration. Node count value cannot be 1. Required for creation

Comment (XiaofeiCao) [reply sdk/cosmosdbforpostgresql/azure-resourcemanager-cosmosdbforpostgresql/src/test/java/com/azure/resourcemanager/cosmosdbforpostgresql/CosmosDBForPostgreSqlManagerTests.java:101]: Got it, thanks.

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**